userServiceSQLHelper.js

```diff
@@ -0,0 +1,28 @@
+function sortBy(sortBy) {
+    if (sortBy == "FirstName") {
+        return "FirstName";
+    } else if (sortBy == "LastName") {
+        return "LastName";
+    } else if (sortBy == "Age") {
+        return "Age";
+    } else if (sortBy == "Phone") {
+        return "Phone";
+    } else if (sortBy == "Email") {
+        return "Email";
+    } else if (sortBy == "Gender") {
+        return "Gender";
+    } else {
+        return "UserID";
+    }
+};
+
+function sortDirection(sortDirection) {
```

Comment:
It have no sense.  `ASC` is default

Reply:
Fixed. Did the same thing as for sortBy.